elide-model-config/src/main/java/com/yahoo/elide/modelconfig/parser/handlebars/HandlebarsHelper.java

```diff
@@ -98,12 +84,17 @@ public String toUpperCase(Object obj) {
     }
 
     /**
-     * Transform string to upper case and replace space with underscore.
-     * @param obj Object representation of the string
-     * @return string converted to upper case and spaces replaced
+     * Capitalize first letter, replace space with underscore and dot with euro sign.
+     * @param obj Object representation of the string.
+     * @param toUpperCase Change case to upper for converted string.
+     * @return converted string.
      */
-    public String toUpperCaseReplaceSpace(Object obj) {
-        return toUpperCase(obj).replace(SPACE, UNDERSCORE);
+    public String createSecurityIdenitfier(Object obj, boolean toUpperCase) {
+        String id = capitalizeFirstLetter(obj.toString()).replace(SPACE, UNDERSCORE).replace(DOT, DOLLAR_SIGN);
```

Comment:
Replacing `.` with `$` sign so that dynamic compiler doesn't complain about file and class name.